src/PlugKeyRing/index.ts

```diff
@@ -412,6 +429,15 @@ class PlugKeyRing {
       return false;
     }
   }
+
+  // Utils
+  private getMainAccountId = (): string => {
+    const  { wallets } = this.state;
+    const mainAccount = Object.values(wallets).find(
+      (wallet) => wallet.orderNumber === 0);
+
+    return mainAccount?.walletId || this.currentWalletId;
```

Comment:
why would it fail?

Reply:
I can't however typescript has no way to know that there will always be a wallet with `orderNumber` equal to `0`

Reply:
you can force the type with a cast or using `!`